packages/renderer/src/lib/ContainerList.svelte

```diff
@@ -520,7 +573,11 @@ function errorCallback(container: ContainerInfoUI, errorMessage: string): void {
                 </td>
                 <td class="flex flex-row justify-center h-12">
                   <div class="grid place-content-center ml-3 mr-4">
-                    <StatusIcon icon="{ContainerIcon}" status="{container.state}" />
+                    {#if iconClass(container)}
+                      <StatusIcon icon="{iconClass(container)}" status="{container.state}" />
+                    {:else}
+                      <StatusIcon icon="{ContainerIcon}" status="{container.state}" />
```

Comment:
wondering if it should be part of the container object ?

like `container.icon`